src/server/utils/onModuleLoad.js

```diff
@@ -91,6 +87,47 @@ export default function onModuleLoad({
       pwa,
       errorPageUrl,
       dnsCache,
+      redirectAllowList,
```

Comment:
```suggestion
      redirectAllowList = [],
```

Reply:
Going to leave this as-is as it follows the convention of the other fields

Reply:
okay, i think it would reduce the complexity so no need to refactor that block out.